src/test/java/LoginTests.java

```diff
@@ -6,23 +8,25 @@
 
 import java.time.Duration;
 
+
+
 public class LoginTests extends BaseTest {
+
     @Test
-    public void LoginEmptyEmailPasswordTest() {
+    public void loginValidEmailPasswordTest() {
 
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        loginPage.login();
 
-//      Create fork : Make changes : Add Comment : Save
+  
+       /* loginPage.provideEmail("tesfaye.abagaz@testpro.io");
+        loginPage.providePassword("te$t$tudent");
+        loginPage.clickSubmit();*/
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.